packages/conf-libmosquitto/conf-libmosquitto.1/opam

```diff
@@ -1,15 +1,20 @@
-opam-version: "1"
+opam-version: "1.2"
 maintainer: "Markus W. Weissmann <markus.weissmann@in.tum.de>"
 authors: [ "Markus W. Weissmann <markus.weissmann@in.tum.de>" ]
 homepage: "https://mosquitto.org/"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: [ "Eclipse Public License v1.0" "Eclipse Distribution License 1.0" ]
 build: [
-  ["ls" "/usr/include/mosquitto.h" {os != "darwin"}]
-  ["brew" "ls" "--versions" "mosquitto" {os = "darwin"}]
+  ["true"] {os = "centos"}
```

Comment:
Please don't put this line. If I manually install libmosquitto on centos and then this package, I'd like for it to do a proper install.

Available below should be enough. I'll merge this with the failure in the CI and Travis (ubuntu 14.04 does not provide libmosquitto as well). We cannot avoid such failures I believe